Content page

DorienHuijser opened the pull request:
Made a start with 2 versions of the content page. The second one is similar to the speaker page, but is not divided into columns yet. I'm not sure how to fix that.

If we're good using the 2020 way, we can also use that :)

DorienHuijser committed:
add include to content2021.html

DorienHuijser committed:
create working but not pretty version of content 2021 page

DorienHuijser commented:
Edit: we also need to do something with the `linkname` in the yml file, that one is not used yet at the moment.  I suspect we may be able to use the linkname to link from the program page :)
Another edit: linkname is now integrated as div id for each card

Remi-Gau committed:
lint file

DorienHuijser committed:
Merge pull request #2 from Remi-Gau/remi-linting

Some light linting

DorienHuijser committed:
make text bigger and add back to top icon

DorienHuijser committed:
remove ugly version + add linkname as card id

sofievdbos commented:
Looks greaaaaaaaaaaaaaaaat! ❤️ 
Thanks @DorienHuijser & @Remi-Gau!!!

**One remark**: 
We have three slide links on the program page & by now we probably have more that were shared with us. What about changing the "Slides" header on the program page to "Content" & link the content links to the respective content cards?

**Edit**: 
Checked the links, they all work! 🎉 Just saw that cuz Stephan has tooooo many links 🙈 the second "Repository" name doesn't fit in the box.

sofievdbos commented on _data/content2021.yml:
```diff
@@ -0,0 +1,228 @@
+
+  - name: "Kayla Green"
+    linkname: "kayla"
+    image: "kayla.jpg"
```
```suggestion
    image: "kayla.JPG"
```

sofievdbos commented on _data/content2021.yml:
```diff
@@ -0,0 +1,228 @@
+
+  - name: "Moss Zhao & Patricia Clement"
+    linkname: "podcast"
+    image: "moss.jpg"
```
```suggestion
    image: {"moss.jpg","you.png"}
```

sofievdbos committed:
Apply suggestions from code review

- fix img Kayla Green
- add img for Patricia Clement (so you can see there are two speakers)

DorienHuijser commented:
Agreed, let's change the "Slides" header to "Content" on the program page (I can do this later today)
"the second "Repository" name doesn't fit in the box." -> Mehh maybe change the name to Repo instead of Repository so that it fits? :D

sofievdbos commented:
🚀 & 🚀 => sounds superrr!

DorienHuijser committed:
change Slides to Content

DorienHuijser committed:
Change Repository to Repo

DorienHuijser committed:
Add links to content

DorienHuijser committed:
Merge branch 'doriencontent' of https://github.com/DorienHuijser/openmrbenelux.github.io into doriencontent

DorienHuijser commented:
Ok I think most things should work now, incl. the links from the program page to the content page. The only thing we need to do is now add the slides from speakers to the OSF page and then paste the links in the `content2021.yml` :D
Or am I still missing something else?

sofievdbos commented:
You are so fast! Thought to surprise you by adding the links to the program page, but alas, you were faster. 😆
Can confirm that the links work! Sounds like the OSF is the only thing left indeed (except for maybe cross-checking titles on the content page with the ones on the program page, but that's a detail).

DorienHuijser committed:
get consistent talk titles

DorienHuijser committed:
get consistent talk titles

DorienHuijser committed:
add osf links part 1

sofievdbos reviewed (approved):
LGTM! (first time I use this abbreviation 😁)
